common/protob/messages-management.proto

```diff
@@ -181,6 +182,7 @@ message ApplySettings {
     optional SafetyCheckLevel safety_checks = 9;  // Safety check level, set to Prompt to limit path namespace enforcement
     optional bool experimental_features = 10;  // enable experimental message types
     optional bool hide_passphrase_from_host = 11;  // do not show passphrase coming from host
+    optional uint32 brightness = 12;  // display brightness, 0 = select on device
```

Comment:
Not right now, but will be in the future.

Reply:
so keeping it as is?

Reply:
i still kind of think that this should be a separate `SetBrightness` message
for all settings, you **send** the value from host, and **confirm** on Trezor

But for brightness, you may **suggest** a value from host, but you're displaying some sort of _interactive control_ on Trezor for changing or confirming that value. I don't expect there to ever be a dialog "set brightness to 15? yes/no" on Trezor, am I right @Hannsek ?

could go either way but I'd lean towards treating brightness as its own workflow separate from the "apply any number of settings from this list" workflow. It's not like it is reusing any code from `apply_settings` either. what do yall think?

Reply:
n.b., having a `SetBrightness` message allows you to make it
```protobuf
message SetBrightness {
    optional uint32 value = 1;  // if not specified, let the user choose
}
```
which doesn't work for settings and you have to use `0` as a special value

Reply:
The ability to have optional value provided and no special meaning for 0 is compelling argument for me.

Reply:
It might be like that. It also might be the same as threshold -> host sends a message with specified value, but you can change it on device as well. But we don't know yet.

Reply:
From the perspective of future extensibility, having separate message is clearly better.

Done here: https://github.com/trezor/trezor-firmware/commit/bf81a08d03d32c295d44d49c147d07de0433350b